BEST_PRACTICES.md

```diff
@@ -195,3 +196,38 @@ var photo: Photo? {
 Completion blocks in networking calls should be returned in the main thread.
 
 Completion blocks should contain the `error` instead of success/failure blocks.
+
+## Theming
+
+Use a `Theme.swift` that contains extensions for theming your app. When adding color extensions avoid adding the `Color` postfix to your color methods.
+
+For example:
+
+```swift
+import Hex
+import UIKit
+
+extension UIColor {
+    class func brightPurple() -> UIColor {
```

Comment:
Yes! That could totally work.

```
callToActionNormal
callToActionSelected
callToActionHighlighted
```

We just need to be careful to not have one method per UI element.